### PR TITLE
Fix key registration

### PR DESCRIPTION
**Pull request by zevlee**
Fixes https://github.com/nextcloud/twofactor_webauthn/issues/233
Signed-off-by: Zev Lee <60147316+zevlee@users.noreply.github.com>

### CODE REVIEW

**Comment by zevlee** [comment]
Let me know if there are any changes I need to make.

File: lib/Migration/Version010000Date20221227080000.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright 2022 Christoph Wurst <christoph@winzerhof-wurst.at>
```
Comment (ChristophWurst) [inline lib/Migration/Version010000Date20221227080000.php:6]: It wasn't me :)

**Review by ChristophWurst — COMMENTED** [review]
The only thing missing is bumping the version in `info.xml`. Change https://github.com/nextcloud/twofactor_webauthn/blob/9c60f8769979e80e9d0d1997c3a2b74f4e9c2b3b/appinfo/info.xml#L7 from .0 to .1

File: lib/Migration/Version010000Date20221227080000.php (hunk 1 above)
Comment (zevlee) [reply lib/Migration/Version010000Date20221227080000.php]: Ah, thank you for pointing that out.

**Comment by zevlee** [comment]
> The only thing missing is bumping the version in `info.xml`. Change
> 
> https://github.com/nextcloud/twofactor_webauthn/blob/9c60f8769979e80e9d0d1997c3a2b74f4e9c2b3b/appinfo/info.xml#L7
> 
> from .0 to .1

Did you want the line to read

```xml
<version>1.1.1-alpha.0</version> 
```

or

```xml
<version>1.1.0-alpha.1</version> 
```

**Comment by ChristophWurst** [comment]
The second one :)

**Review by ChristophWurst — APPROVED** [review]
Looks good

**Comment by zevlee** [comment]
It looks like there are some workflows that need to be run before this PR can be merged. Is there anything I need to do on my end?

**Review by ChristophWurst — APPROVED** [review]

File: lib/Migration/Version010000Date20221227080000.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		$registrationsTable = $schema->getTable('twofactor_webauthn_regs');
+		/**
+		* Fixes https://github.com/nextcloud/twofactor_webauthn/issues/233
+		*/
```
Comment (ChristophWurst) [inline lib/Migration/Version010000Date20221227080000.php:48]:
```suggestion
		 * Fixes https://github.com/nextcloud/twofactor_webauthn/issues/233
		 */
```

**Review by ChristophWurst — COMMENTED** [review]
Nitpick

When you *amend* your commit, change the commit message from ``Fix key registration`` to ``fix(database schema): Fix public key column size`` 

Ref https://www.conventionalcommits.org/en/v1.0.0/

**Comment by zevlee** [comment]
I changed the commit message as you requested.

File: lib/Migration/Version010000Date20221227080000.php (hunk 2 above)
Comment (zevlee) [reply lib/Migration/Version010000Date20221227080000.php]: I applied this change and made it a part of the commit.

**Comment by zevlee** [comment]
My apologies, it seems that I forgot to add a blank line before the footer of the commit message. It should be fixed now.

**Comment by zevlee** [comment]
Not sure why the commit message check is failing. Perhaps the scope must be limited to a single word? i.e. `fix(database): Fix public key column size`

**Review by st3iny — APPROVED** [review]

**Pull request merged**